profile.html

```diff
@@ -17,54 +17,55 @@
     <!-- Local styles -->
     <link rel="stylesheet" href="styles/common_styles.css">
     <link rel="stylesheet" href="styles/profile_styles.css">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 </head>
 
 <body>
-    <div id="wrapper">
-
+    <main id="wrapper">
         <!--User image, name, edit info button.-->
-        <section id="user_pic_block" class="bg-darker container">
+        <section id="user_pic_block" class="bg-darker rounded">
             <img class="img-avatar img-thumbnail" src="https://dummyimage.com/256/256" alt="Profile Picture">
             <div>
                 <fieldset id="nameField" disabled>
                     <input hidden id="nameInput" type="text" class="form-control">
                     <h1 class="text-light h3" id="nameText"></h1>
                 </fieldset>
-                    <h4 id="emailText"></h4>
-                    <img id="editInfo" src="icons/edit-2.svg#icon" onclick="editProfile()" class="edit-icon" alt="Edit">  
-                    <img hidden id="saveInfo" src="icons/edit-2.svg#icon" onclick="saveProfile()" class="edit-icon" alt="Save">  
-                    <!-- <svg id="saveInfo" onclick="saveProfile()" class="edit-icon" aria-hidden="true" focusable="false">
+                <h4 id="emailText"></h4>
+                <!-- <img id="editInfo" src="icons/edit-2.svg#icon" onclick="editProfile()" class="edit-icon" alt="Edit">   -->
+                <img hidden id="saveInfo" src="icons/edit-2.svg#icon" onclick="saveProfile()" class="edit-icon"
+                    alt="Save">
+                <!-- <svg id="saveInfo" onclick="saveProfile()" class="edit-icon" aria-hidden="true" focusable="false">
                         <use xlink:href="icons/edit-2.svg#icon"></use>
                     </svg> -->
-                </div>
-                <img src="icons/edit-2.svg#icon" alt="test">
-          
+            </div>
+
```

Comment:
No more edit button?

Reply:
Add it back when you do image uploading, I don't want to deal with SVG headaches while the feature isn't even being worked on :P